<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>68</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>OSD Notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/res/logo/logo.png</normaloff>:/res/logo/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionInsertImage"/>
   <addaction name="actionTable"/>
   <addaction name="actionTransIncrease"/>
   <addaction name="actionTransDecrease"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBarFont">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSmallImage"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionDeleteline"/>
   <addaction name="actionReset"/>
   <addaction name="actionShortcut"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/bold.png</normaloff>:/res/icon/bold.png</iconset>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="toolTip">
    <string>Set current font bold. Ctrl+B</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/italic.png</normaloff>:/res/icon/italic.png</iconset>
   </property>
   <property name="text">
    <string>I</string>
   </property>
   <property name="toolTip">
    <string>Set current font italic. Ctrl+I</string>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/format-underline.png</normaloff>:/res/icon/format-underline.png</iconset>
   </property>
   <property name="text">
    <string>U</string>
   </property>
   <property name="toolTip">
    <string>Set current font underline. Ctrl+U</string>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDeleteline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/format-strike.png</normaloff>:/res/icon/format-strike.png</iconset>
   </property>
   <property name="text">
    <string>abc</string>
   </property>
   <property name="toolTip">
    <string>Set current font deleteline. Ctrl+D</string>
   </property>
   <property name="font">
    <font>
     <strikeout>true</strikeout>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/reset.png</normaloff>:/res/icon/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset font format. Ctrl+R</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInsertImage">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/insert_img.png</normaloff>:/res/icon/insert_img.png</iconset>
   </property>
   <property name="text">
    <string>InsertImage</string>
   </property>
   <property name="toolTip">
    <string>Insert image to the file.</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/open_file.png</normaloff>:/res/icon/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a file. Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/save_file.png</normaloff>:/res/icon/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the file. Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/new_file.png</normaloff>:/res/icon/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new file. Ctrl+N</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/view_off.png</normaloff>
     <normalon>:/res/icon/view_on.png</normalon>:/res/icon/view_off.png</iconset>
   </property>
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View mode, readonly. Ctrl+Shift+V</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+V</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/alway_on_top.png</normaloff>:/res/icon/alway_on_top.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="toolTip">
    <string>Always on top. Ctrl+Shift+T</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionTransIncrease">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/opacity_increase.png</normaloff>:/res/icon/opacity_increase.png</iconset>
   </property>
   <property name="text">
    <string>T+</string>
   </property>
   <property name="toolTip">
    <string>Increase transparency.</string>
   </property>
  </action>
  <action name="actionTransDecrease">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/opacity_decrease.png</normaloff>:/res/icon/opacity_decrease.png</iconset>
   </property>
   <property name="text">
    <string>T-</string>
   </property>
   <property name="toolTip">
    <string>Decrease transparency.</string>
   </property>
  </action>
  <action name="actionSmallImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/img_origin.png</normaloff>
     <normalon>:/res/icon/img_small.png</normalon>:/res/icon/img_origin.png</iconset>
   </property>
   <property name="text">
    <string>SmallImage</string>
   </property>
   <property name="toolTip">
    <string>Set all images to small or original size.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/about.png</normaloff>:/res/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About the application.</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/save_file_as.png</normaloff>:/res/icon/save_file_as.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="toolTip">
    <string>Save as others.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionShortcut">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/shortcut.png</normaloff>:/res/icon/shortcut.png</iconset>
   </property>
   <property name="text">
    <string>Shortcut</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icon/table.png</normaloff>:/res/icon/table.png</iconset>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
